projects/isy-angular-widgets-demo/src/app/feature/isy-angular-components/isy-angular-components.component.ts

```diff
@@ -0,0 +1,45 @@
+import {Component} from '@angular/core';
+import {FormBuilder, FormGroup, ReactiveFormsModule, Validators} from '@angular/forms';
+import {FormWrapperComponent} from '@isy-angular-widgets/form-wrapper/form-wrapper.component';
+import {TranslateModule} from '@ngx-translate/core';
+import {FormControlPipe} from '@isy-angular-widgets/pipes/form-control.pipe';
+import {HauptfensterModule, IncompleteDateModule, SeitentoolbarComponent} from '@isy-angular-widgets/public-api';
+import {initializedPerson} from '../objekt-anzeigen/data';
+import {InputCharDirective} from '@isy-angular-widgets/input-char/directives/input-char.directive';
+import {InputTextModule} from 'primeng/inputtext';
+import {ButtonModule} from 'primeng/button';
+import {MenuModule} from 'primeng/menu';
+import {PanelModule} from 'primeng/panel';
+
+@Component({
+  selector: 'demo-isy-angular-components',
+  standalone: true,
+  templateUrl: './isy-angular-components.component.html',
+  imports: [
+    FormWrapperComponent,
+    TranslateModule,
+    FormControlPipe,
+    ReactiveFormsModule,
+    IncompleteDateModule,
+    InputCharDirective,
+    InputTextModule,
+    SeitentoolbarComponent,
+    ButtonModule,
+    HauptfensterModule,
+    PanelModule,
+    MenuModule
+  ]
```

Comment:
Imports prüfen, mMn brauchen wir mind. HauptfensterModule, PanelModule, MenuModule nicht mehr
```suggestion
  imports: [
    FormWrapperComponent,
    TranslateModule,
    FormControlPipe,
    ReactiveFormsModule,
    IncompleteDateModule,
    InputCharDirective,
    InputTextModule,
    SeitentoolbarComponent,
    ButtonModule
  ]
```